catalyst/engines/amp.py

```diff
@@ -386,13 +386,13 @@ def pack_checkpoint(
             ``criterion_state_dict``, ``optimizer_state_dict``,
             ``scheduler_state_dict`` keys.
         """
-        checkpoint = {"scaler": self.scaler.state_dict()}
+        kwargs["scaler"] = self.scaler.state_dict()
```

Comment:
What is the point of these changes?

Reply:
We need to transfer kwargs to save extra features like "global_epoch_step" and other. Without them resume option doesn't work.